src/main/java/mate/academy/security/AuthenticationServiceImpl.java

```diff
@@ -28,7 +28,7 @@ public User register(String email, String password) {
         User user = new User();
         user.setEmail(email);
         user.setPassword(password);
-        user.setRoles(Set.of(roleService.getRoleByName("USER")));
+        user.setRoles(Set.of(roleService.getRoleByName("ADMIN")));
```

Comment:
all users that we register should be admins?

Reply:
changed